docs/hass-image-install.md

```diff
@@ -59,15 +66,29 @@ Install [Raspberry Pi Imager](https://www.raspberrypi.com/software/) on your com
 <robo-wiki-picture src="home-assistant/insert-sd-card.gif" alt="insert SD card" />
 
 
-Run the Raspberry Pi Imager program. Choose required image as the operating system and ensure to select your SD card from the storage dropdown menu. In settings provide your Wi-Fi, password, choose your country from drop-down list and then `Write!` image. 
-
-<robo-wiki-video autoplay loop controls :videos="[{src: 'https://crustipfs.info/ipfs/QmVgex7Aw9sH97kDbbwaHkbcHZRu8xWruh5a7hfnMstNrV', type:'mp4'}]" />
+Run the Raspberry Pi Imager program. Choose required image as the operating system and ensure to select your SD card from the storage dropdown menu. 
+In settings:
+- Set username and password (save default username "pi" to be easy to remember),  
+- provide your Wi-Fi name and password, 
+- choose your country from drop-down list
+and then `Write!` image. 
```

Comment:
maybe just `Write`(without !) will be enough?

Reply:
fixed https://github.com/airalab/robonomics-wiki/pull/632/commits/7162715d12486f2d2066492cd74dc2e3ea69dc02